src/main/java/blob/common/Messages.java

```diff
@@ -0,0 +1,36 @@
+package blob.common;
+
+/**
+ * Class containing the reusable messages to print to user
+ */
+public class Messages {
+    public static final String MESSAGE_WELCOME_1 = "Hello... me Blob...";
```

Comment:
These variables are very clearly named and is descriptive, which helps readers to better understand your code.